source/TLS.rst

```diff
@@ -59,71 +62,256 @@ The steps to install the *cert-manager* are the following:
+   $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.7/deploy/manifests/cert-manager.yaml
 
 After the installation, you can verify the *cert-manager* by running the following command:
 
-::
-  
-  kubectl get pods -n cert-manager
+.. code:: bash
+
+   $ kubectl get pods -n cert-manager
 
 The result should display the *cert-manager* and webhook active and running.
 
+.. _tls.certs.manual:
+
 Generate certificates manually
 ==============================
 
 To generate certificates manually, follow these steps:
 
 1. Provision a Certificate Authority (CA) to generate TLS certificates
+
 2. Generate a CA key and certificate file with the server details
+
 3. Create the server TLS certificates using the CA keys, certs, and server
    details
 
 The set of commands generate certificates with the following attributes:
 
 *  ``Server-pem`` - Certificate
+
 *  ``Server-key.pem`` - the private key
+
 *  ``ca.pem`` - Certificate Authority
 
 You should generate certificates twice: one set is for external communications,
 and another set is for internal ones. A secret created for the external use must
 be added to ``cr.yaml/spec/secretsName``. A certificate generated for internal
 communications must be added to the ``cr.yaml/spec/sslInternalSecretName``.
 
-::
-
-  cat <<EOF | cfssl gencert -initca - | cfssljson -bare ca
-  {
-    "CN": "Root CA",
-    "key": {
-      "algo": "rsa",
-      "size": 2048
-    }
-  }
-  EOF
-
-  cat <<EOF | cfssl gencert -ca=ca.pem  -ca-key=ca-key.pem - | cfssljson -bare server
-  {
-    "hosts": [
-      "${CLUSTER_NAME}-proxysql",
-      "*.${CLUSTER_NAME}-proxysql-unready",
-      "*.${CLUSTER_NAME}-pxc"
-    ],
-    "CN": "${CLUSTER_NAME}-pxc",
-    "key": {
-      "algo": "rsa",
-      "size": 2048
-    }
-  }
-  EOF
-
-  kubectl create secret generic my-cluster-ssl --from-file=tls.crt=server.pem --
-  from-file=tls.key=server-key.pem --from-file=ca.crt=ca.pem --
-  type=kubernetes.io/tls
+.. code:: bash
+
+   $ cat <<EOF | cfssl gencert -initca - | cfssljson -bare ca
+   {
+     "CN": "Root CA",
+     "key": {
+       "algo": "rsa",
+       "size": 2048
+     }
+   }
+   EOF
+
+   $ cat <<EOF | cfssl gencert -ca=ca.pem  -ca-key=ca-key.pem - | cfssljson -bare server
+   {
+     "hosts": [
+       "${CLUSTER_NAME}-proxysql",
+       "*.${CLUSTER_NAME}-proxysql-unready",
+       "*.${CLUSTER_NAME}-pxc"
+     ],
+     "CN": "${CLUSTER_NAME}-pxc",
+     "key": {
+       "algo": "rsa",
+       "size": 2048
+     }
+   }
+   EOF
+
+   $ kubectl create secret generic my-cluster-ssl --from-file=tls.crt=server.pem --
+   from-file=tls.key=server-key.pem --from-file=ca.crt=ca.pem --
+   type=kubernetes.io/tls
+
+.. _tls.cets.update:
+
+Update certificates
+===================
+
+If a :ref`cert-manager<tls.certmanager>` is used, it should take care of
+updating the certificates. If you :ref:`generate certificates manually<tls.certs.manual>`,
+you are also in charge for updating them.
```

Comment:
"in charge" caught my eye. Not sure why, but I think there should be better wording. Not mandatory to be changed, but pls give it a thought.